src/include/sof/audio/kpb.h

```diff
@@ -15,8 +15,16 @@
 struct comp_buffer;
 
 /* KPB internal defines */
+
+#ifdef CONFIG_TIGERLAKE
```

Comment:
It should be #if CAVS_VERSION == CAVS_VERSION_2_5

Reply:
Thing is, this change doesn't have much to do with the cavs version, rather very specific hardware capabilities.
So, unless cAVS 2.5 guarantees a certain memory configuration to be present, I'd rather keep it as is.

Reply:
@slawblauciak not sure if we have memory constraints for TGL-H also?

Reply:
No clue, I'm afraid.

Reply:
Ok, that makes sense 👍